src/material/datepicker/date-range-input-parts.ts

```diff
@@ -54,7 +56,7 @@ export interface MatDateRangeInputParent {
  * to the parts without circular dependencies.
  */
 export const MAT_DATE_RANGE_INPUT_PARENT =
-    new InjectionToken<MatDateRangeInputParent>('MAT_DATE_RANGE_INPUT_PARENT');
+    new InjectionToken<MatDateRangeInputParent<any>>('MAT_DATE_RANGE_INPUT_PARENT');
```

Comment:
can we use `unknown` or does that cause issues?

Reply:
It doesn't seem to throw anything, but I'm guessing that it's because the compiler isn't validating it. I'll change it to `unknown`.